### PR TITLE
Add reference to the gather-town space

### PR DESCRIPTION
**Pull request by jfy133**
image

### CODE REVIEW

**Comment by mashehu** [comment]
I am not sure, if this is visible enough... I would make it also a whole section after the youtube section.

**Comment by renbot-bio** [comment]
I think badge would do

**Comment by jfy133** [comment]
My idea of doing as such was we want to somewhat restrict access (as there isn't people monitoring all the time like on slack). 

The badge doesn't help because don't want to make direct access. Although maybe as it is password protected maybe it doesn't matter :thinking: 

What do you think is better? Is a badge OK?

**Comment by renbot-bio** [comment]
> I am not sure, if this is visible enough... I would make it also a whole section after the youtube section.

OK, agree with Mattias's suggestion.

**Comment by ewels** [comment]
+1 for a section

**Comment by jfy133** [comment]
Done. SVG icon for gather.town unfortunately doens't exist, so removed it :-1: 

@renbot-bio @ewels @mashehu

File: public_html/join.php [hunk 1]
```diff
@@ -108,4 +107,17 @@
 <script src="https://apis.google.com/js/platform.js"></script>
 <div class="g-ytsubscribe" data-channelid="UCfdY8g6eA6IuYZ8oyyNsabQ" data-layout="full" data-count="default"></div>
 
+<h1 id="Gather-town" class="mt-5">
+  Gather Town
+  <a href="#twitter" class="header-link"><span class="fas fa-link" aria-hidden="true"></span></a>
```
Comment (mashehu) [inline public_html/join.php:112]:
```suggestion
  <a href="#gather-town" class="header-link"><span class="fas fa-link" aria-hidden="true"></span></a>
```

File: public_html/join.php [hunk 2]
```diff
@@ -108,4 +107,17 @@
 <script src="https://apis.google.com/js/platform.js"></script>
 <div class="g-ytsubscribe" data-channelid="UCfdY8g6eA6IuYZ8oyyNsabQ" data-layout="full" data-count="default"></div>
 
+<h1 id="Gather-town" class="mt-5">
```
Comment (mashehu) [inline public_html/join.php:110]:
```suggestion
<h1 id="gather-town" class="mt-5">
```

File: public_html/join.php [hunk 3]
```diff
@@ -108,4 +107,17 @@
...
+  Gather Town
+  <a href="#twitter" class="header-link"><span class="fas fa-link" aria-hidden="true"></span></a>
+</h1>
+<p>nf-core has a virtual meeting workspace on <a href="https://gather.town/app/b2RCFGS2cIGusuNi/nf-core-office">gather.town</a> for face-to-face collaboration.
```
Comment (mashehu) [inline public_html/join.php:114]:
```suggestion
<p>nf-core has a virtual meeting workspace on <a href="https://gather.town/app/b2RCFGS2cIGusuNi/nf-core-office">gather.town</a> for collaboration. 
```

**Review by mashehu — APPROVED** [review]

**Comment by jfy133** [comment]
Replaced with one @mashehu made.

@renbot-bio if you approve we can merge in (just in case you felt we needed any other info)

**Comment by jfy133** [comment]
Sorry found indeed! Brain not at 100 after travelling for 16 hours yday :face_with_spiral_eyes:

File: public_html/join.php [hunk 4]
```diff
@@ -36,7 +36,11 @@
   <a href="https://www.youtube.com/c/nf-core" class="mb-2 btn btn-lg btn-danger" style="background-color: #EA3C1E;">
     <i class="fab fa-youtube"></i> nf-core on YouTube
   </a>
+  <a href="https://gather.town/app/b2RCFGS2cIGusuNi/nf-core-office" class="mb-2 btn btn-lg btn-info" style="background-color: #4358D8;">
+  <i class="fa-duotone fa-video"></i> nf-core on gather.town
```
Comment (ewels) [inline public_html/join.php:40]: Are we going to drop this button?

Comment (jfy133) [reply public_html/join.php]: Yup, done

**Review by ewels — APPROVED** [review]
👏🏻

**Pull request merged**